server/api/api.go

```diff
@@ -1063,6 +1064,66 @@ func (a *API) handleGetUser(w http.ResponseWriter, r *http.Request) {
 	auditRec.Success()
 }
 
+func (a *API) handleGetUsersByTeamIDAndIds(w http.ResponseWriter, r *http.Request) {
```

Comment:
example of inconsistent case `ID` vs `Id` in function signature

Reply:
Okay, Got it.